scenario_gym/viewer/opencv.py

```diff
@@ -230,6 +231,21 @@ def get_center_pose(self, state: State, e_ref: str) -> np.ndarray:
             ego_pose = np.zeros(6)
         return ego_pose
 
+    @property
+    def entity_colors(self) -> Dict[int, Color]:
+        """Store the entity color dict."""
+        if self._entity_colour_dict is None:
+            if self._state is None:
```

Comment:
Why do we make `_state` an attribute of the viewer instead of passing it as an argument? It could be passed here right?

Reply:
yes we could - I used it because it was already initialised [here](https://github.com/driskai/scenario_gym/blob/849f745aa3dc37d9852830d7d47c00484367b04c/scenario_gym/viewer/opencv.py#L155). And like that we can make `entity_colors` a property, but we can easily change it to `get_entity_colors` or something.

Reply:
Ah so it was there but unused? I'm not sure whether it should be or not. I guess its fine to leave.